src/app/dim-ui/PressTip.tsx

```diff
@@ -259,48 +259,73 @@ export function PressTip(props: Props) {
   const timer = useRef<number>(0);
   const touchStartTime = useRef<number>(0);
   const ref = useRef<HTMLDivElement>(null);
+  const startEvent = useRef<'pointerdown' | 'pointerenter'>();
   const [open, setOpen] = useState<boolean>(false);
 
-  const closeToolTip = useCallback(() => {
+  const closeToolTip = useCallback((e: React.PointerEvent) => {
+    // Ignore events that aren't paired up
+    if (
+      !startEvent.current ||
+      (e.type === 'pointerup' && startEvent.current === 'pointerenter') ||
+      (e.type === 'pointerleave' && startEvent.current === 'pointerdown')
+    ) {
+      return;
+    }
     setOpen(false);
     clearTimeout(timer.current);
-    timer.current = 0;
+    // Don't clear these for 500ms to give the click event time to suppress - pointerup fires before click
+    setTimeout(() => {
+      timer.current = 0;
+      startEvent.current = undefined;
+    }, 500);
   }, []);
 
   const hover = useCallback((e: React.PointerEvent) => {
     e.preventDefault();
+    // If we're already hovering, don't start hovering again
+    if (
+      startEvent.current &&
+      // Safari, at least, fires both pointerenter and pointerdown at the same time. We want the pointerdown event.
+      !(startEvent.current === 'pointerenter' && performance.now() - touchStartTime.current < 10)
```

Comment:
In my testing these fired simultaneously, so the 10ms fudge factor here should be plenty.